mutation-testing/pr-differences/action.yml

```diff
@@ -0,0 +1,127 @@
+name: PR Differences Mutants
+
+inputs:
+  shard:
+    description: "The number of the shard to run (`-1` if ran without shards)"
+    required: false
+    default: -1
+  package-dimension:
+    description: "The dimension of the package. `big` for `stacks-node` and `stackslib`, `small` for others"
+    required: true
+
+runs:
+  using: "composite"
+
+  steps:
+    # Cleanup Runner
+    - name: Cleanup Runner action repo
+      id: runner_cleanup
+      uses: stacks-network/actions/cleanup/disk@main
+
+    # Checkout the stacks-core code
+    - name: Checkout the stacks core repo
+      id: git_checkout_stacks_core
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Relative diff
+      shell: bash
+      run: |
+        git diff origin/${{ github.base_ref }}.. > git.diff
+
+    # Checkout the actions code
+    - name: Get the action repo code
+      id: git_checkout_actions
+      uses: actions/checkout@v3
+      with:
+        repository: stacks-network/actions
+        ref: feat/mutation-testing
+        path: ./actions-repo
+
+    - name: Copy the git diff file to scripts folder
+      shell: bash
+      run: |
+        mv git.diff ./actions-repo/mutation-testing/shell-scripts/
+
+    - name: Install cargo mutants crate
+      shell: bash
+      run: cargo install --version 23.12.2 cargo-mutants
+
+    - name: Remove deleted file lines from git.diff file
+      shell: bash
+      run: ./remove-deleted-file-lines.sh
+      working-directory: actions-repo/mutation-testing/shell-scripts
+
+    - name: Split diffs into big and small packages and create regex patterns for them
+      shell: bash
+      run: |
+        cargo mutants --in-diff git.diff --list > all_mutants.txt
```

Comment:
be sure to check return codes after something is run - no need to continue if the command has errored out.

Reply:
That specific command on L59 is checked afterwards, in the newly added file verification (in the case the command errors, no output is being written to `all_mutants.txt`, so it is an empty file - https://github.com/stacks-network/actions/blob/feat/mutation-testing/mutation-testing/pr-differences/action.yml#L62-L66).
The other commands have their exit codes captured and sent to the parent workflow for them to be checked - and they are the last ones running, so the job finishes after they are run anyways.